examples/NavigationPlayground/package.json

```diff
@@ -12,13 +12,13 @@
   },
   "dependencies": {
     "expo": "^24.0.2",
-    "react": "16.0.0",
-    "react-native": "^0.51.0",
+    "react": "16.2.0",
+    "react-native": "^0.52.0",
```

Comment:
can you either undo this change, or make the same version change for the redux example and the main library `package.json`? I think we should keep all the versions in sync